java/src/main/java/org/eclipse/ditto/client/configuration/WebSocketMessagingConfiguration.java

```diff
@@ -88,6 +88,7 @@ public boolean isReconnectEnabled() {
         private ProxyConfiguration proxyConfiguration;
         private TrustStoreConfiguration trustStoreConfiguration;
 
+        @Deprecated
```

Comment:
I think we shouldn't deprecate this method. If we introduce a V3 one day, we need it again. We should rather deprecate/remove the call in `DittoClientUsageExamples.createMessagingProvider`.